src/sentry/models/event.py

```diff
@@ -619,6 +619,7 @@ class Event(EventCommon, Model):
         ref_func=lambda x: x.project_id or x.project.id,
         ref_version=2,
         wrapper=EventDict,
+        skip_nodestore_save=True,
```

Comment:
Is this supposed to stay ? If yes what's a case we would pass False ?

Reply:
This flag temporarily allows for different behaviour on NodeField. In the RawEvent model we still want to perform nodestore save, for Event we do not. When we (eventually) completely remove this NodeField it won't be needed.